packages/amplify_analytics_pinpoint/CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## 0.2.0 (2021-06-30)
+
+### Features
+
+- feat(amplify_analytics_pinpoint): Null safety (#483)
+- feat: Null safety core (#492)
+
+### Chores
+
+- chore: Null safety master rebase (#676)
```

Comment:
*nit:* I would probably omit this. I can't think of a scenario where this info would be particularly useful in a change log. I can see the counter argument of wanting to include it for completeness though.

Reply:
Normally I'd agree, but this was such a massive change I'd err on leaving it in?